lib/src/components/NxProgressBar/NxProgressBar.scss

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-progress-bar {
+  display: flex;
+  flex-direction: column;
+  gap: var(--nx-spacing-2x);
+  margin: var(--nx-spacing-6x) 0;
+}
+
+.nx-progress-bar--inline-counter {
+  flex-direction: row;
+}
+
+.nx-progress-bar--inline {
+  display: inline-flex;
+  margin: 0;
+}
+
+.nx-progress-bar__counter .nx-counter {
+  margin: 0;
+}
+
+.nx-progress-bar--success {
+  color: var(--nx-swatch-green-25);
+
+  .nx-counter {
+    border-color: var(--nx-swatch-green-25);
+    background-color: var(--nx-swatch-green-95);
+  }
+}
+
+.nx-progress-bar--error {
+  color: var(--nx-swatch-red-40);
+
+  .nx-counter {
+    border-color: var(--nx-swatch-red-40);
+    background-color: var(--nx-swatch-red-95);
+  }
+}
+
+.nx-progress-bar__label {
+  align-items: center;
+  display: flex;
+  gap: var(--nx-spacing-2x);
+  grid-area: label;
```

Comment:
What's this for?

Reply:
> What's this for?

Ops, removed: 038fa7afcecbb1202fa5858f30d2bebf690f2cd0